sentry/src/main/java/io/sentry/event/interfaces/SentryStackTraceElement.java

```diff
@@ -13,6 +18,7 @@
     private final Integer colno;
     private final String absPath;
     private final String platform;
+    private final Map<String, Object> vars;
```

Comment:
`locals` seems like a more specific/appropriate name, but not sure if that's excessively specific.